drivers/mssqql.go

```diff
@@ -275,19 +276,36 @@ func (db *MSSQL) GetRecords(database, table, where, sort string, offset, limit i
 		if err := rows.Scan(rowValues...); err != nil {
 			return nil, 0, err
 		}
-
+		// Get column types to identify UNIQUEIDENTIFIER
+		columnTypes, err := rows.ColumnTypes()
+		if err != nil {
+			return nil, 0, err
+		}
 		var row []string
-		for _, col := range rowValues {
+		for i, col := range rowValues {
 			if col == nil {
 				row = append(row, "NULL&")
 				continue
 			}
 
-			colval := string(*col.(*sql.RawBytes))
-			if colval == "" {
-				row = append(row, "EMPTY&")
+			rawBytes := *col.(*sql.RawBytes)
+			colType := columnTypes[i].DatabaseTypeName()
+
+			if colType == "UNIQUEIDENTIFIER" {
+				// Convert binary GUID to standard string format
+				guid, err := uuid.FromBytes(rawBytes)
+				if err != nil {
+					return nil, 0, fmt.Errorf("failed to parse GUID: %w", err)
+				}
+				row = append(row, guid.String())
 			} else {
-				row = append(row, string(*col.(*sql.RawBytes)))
+				// Handle non-GUID columns as strings
+				colval := string(rawBytes)
+				if colval == "" {
+					row = append(row, "EMPTY&")
+				} else {
+					row = append(row, colval)
+				}
```

Comment:
This code could be simplified, also the comment about non-GUID is overkill here, so here is my suggestion

```suggestion
				// Handle other columns as strings
				colval := string(rawBytes)
				if colval == "" {
					colVal = "EMPTY&"
				}
				row = append(row, colval)
```